src/components/DocExplorer.js

```diff
@@ -496,7 +496,7 @@ class TypeDoc extends React.Component {
                 {': '}
                 <TypeLink type={field.type} onClick={onClickType} />
                 {
-                  field.isDeprecated &&
+                  field.deprecationReason &&
```

Comment:
Looks like we're no longer passing in `isDeprecated` to the DocExplorer. This was working, but broke in the last release. Nothing else references `isDeprecated` so it wasn't noticed. Everywhere else we're simply interacting with the `deprecationReason` only.

This patch sorts the issue out, but as you noted is probably just a symptom of the actual  issue (eg. we're not passing in _both_ of these field values).

How about I modify this to be `field.isDeprecated || field.deprecationReason` since that's really what we want anyway?